clients/python-wrapper/utests/test_object_io.py

```diff
@@ -38,41 +38,31 @@ def __init__(self) -> None:
         super().__init__(physical_address="physical_address")
 
 
-@contextmanager
-def client_context(monkey):
-    with monkey.context():
-        from lakefs.client import Client
-        clt = Client()
-        storage_config = StorageTestConfig()
-        monkey.setattr(clt, "_storage_conf", storage_config)
-        yield clt
-
-
 @contextmanager
 def readable_object_context(monkey, tmp_path, **kwargs):
-    with lakectl_test_config_context(monkey, tmp_path):
-        with client_context(monkey) as clt:
-            from lakefs.object_io import ReadableObject
-            read_obj = ReadableObject(client=clt, **kwargs)
-            yield read_obj
+    with lakectl_test_config_context(monkey, tmp_path) as client:
+        from lakefs.object_io import ReadableObject
+        conf = lakefs_sdk.Config(version_config=lakefs_sdk.VersionConfig(), storage_config=StorageTestConfig())
+        monkey.setattr(client.DefaultClient, "_server_conf", conf)
+        read_obj = ReadableObject(client=client.DefaultClient, **kwargs)
```

Comment:
Not exactly - it's a tricky part where we reload the client module inside lakectl_test_config_context so that we reset the DefaultClient initialization